UnityProject/Assets/Scripts/Furniture/TableBuilding.cs

```diff
@@ -95,7 +95,8 @@ private void Assemble(HandApply interaction, string tableType, LayerTile layerTi
 			$"You finish assembling the {tableType} table.",
 			$"{interaction.Performer.ExpensiveName()} assembles a {tableType} table.",
 			() => SpawnTable(interaction, layerTile));
-		SoundManager.PlayNetworkedAtPos(soundName, gameObject.WorldPosServer(), 1f, sourceObj: gameObject);
+		// JESTER
+		// SoundManager.PlayNetworkedAtPos(soundName, gameObject.WorldPosServer(), 1f, sourceObj: gameObject);
```

Comment:
Codacy found an issue: [Remove this commented out code.](https://app.codacy.com/gh/unitystation/unitystation/pullRequest?prid=6280689)